retrofit/src/test/java/retrofit2/CallTest.java

```diff
@@ -781,24 +782,24 @@
     Object a = new Object() {
       @Override public String toString() {
         writeCount.incrementAndGet();
-        throw new OutOfMemoryError("Broken!");
+        throw new AssertionError("Broken!");
       }
     };
     Call<String> call = service.postRequestBody(a);
 
     try {
       call.request();
       fail();
-    } catch (OutOfMemoryError e) {
-      assertThat(e).hasMessage("Broken!");
+    } catch (AssertionError expected) {
```

Comment:
nit: we tend to only use `expected` when there is no `catch` body.

Reply:
ah, i didn't realize that, thanks. i'll update to `e`.